docs/admin/addons.rst

```diff
@@ -96,11 +93,11 @@ Bulk edit
 
 .. versionadded:: 3.11
 
-This addon allow to bulk edit flags, labels or state.
+Mass edit flags, labels or state for strings.
```

Comment:
It used to be called mass editing, but was changed to bulk years ago, so the description probably should be consistent with it.

Reply:
Can confirm.